docs/setup-action.md

```diff
@@ -0,0 +1,37 @@
+# Setup polaris action
```

Comment:
Can we add these to `docs/infrastructure-as-code.md`? That way it gets automatically pulled into the docs at polaris.docs.fairwinds.com